dashboard/src/apollo/httpLink.js

```diff
@@ -1,10 +1,34 @@
 import { BatchHttpLink as HttpLink } from "apollo-link-batch-http";
 import doFetch from "/utils/fetch";
+import gql from "graphql-tag";
 
-const httpLink = () =>
+const mutation = gql`
+  mutation SetLocalNetworkOfflineMutation($offline: Boolean!) {
+    setLocalNetworkOffline(offline: $offline) @client
+  }
+`;
+
+const httpLink = ({ getClient }) =>
   new HttpLink({
     uri: "/graphql",
-    fetch: doFetch,
+    fetch: (url, init) =>
+      doFetch(url, init).then(
+        response => {
+          getClient().mutate({
+            mutation,
+            variables: { offline: true },
```

Comment:
Unless I'm mistaking the intent, do we not want to set the state to false if the query was successful?

Reply:
Whoops, yeah that should be false